src/internal_macros.h

```diff
@@ -11,79 +11,79 @@
 #endif
```

Comment:
we should clang-format off this. when you get around to the format PR please make sure this isn't changed.